### PR TITLE
Capstone 1 

### PR DESCRIPTION
**Pull request by MarcoOquendoC**
# [HTML + basic JavaScript Capstone - day 3](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)

## Hi good reviewer,

Have a good day bear

### These are the [changes](https://github.com/MarcoOquendoC/CapstoneAPEC/tree/desktop) proposed to the [Capstone](https://github.com/MarcoOquendoC/CapstoneAPEC/tree/main) project:

For more detail refer to the [README.md](https://github.com/MarcoOquendoC/CapstoneAPEC/blob/desktop/README.md) file.

## Project requirements

- Personalize the content of the page. Choose a topic that is different than the one in the original design.
- Follow these design guidelines:
  - Colors.
  - Typography: font face, size and weight (we suggest using Lato as it is a free font similar to the one used in the original design).
  - Layout: composition and space between elements.
  - The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.

### Build only these 2 pages:

> The home page.
> The about page.
> Each of these pages should have versions for 2 different screen sizes:

> Mobile: up to 768px wide.
> Desktop: 768px or wider.
> Interactions

### Links
- The home page should have a link in the menu to the about page.
- The logo in the header links to the home page.

#### Mobile menu
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
- The mobile menu should have a close (X) button that closes the menu.

#### Dynamic page
- The section "Featured speakers" should be created dynamically in JavaScript.
- Use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

## Original design idea by Cindy Shin in Behance.
- The Creative Commons license of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

### CODE REVIEW

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @MarcoOquendoC, 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

- You have submitted a Pull Request that is not passing the linters test, but don't worry, just work on that and submit your PR again when all three tests are green.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by MarcoOquendoC** [comment]
# Thank you Sebastián @smunozmo,

- Linter errors where corrected.

It was my mistake. 
the issue was:

- url, src, and href to different files had a slash in the begining and in css files need two dots and slash.

facepalm-gif-2

File: styles/index.css [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+
+#home {
+  padding-top: 4rem;
+  background-image: url("../assets/images/background1.png");
+}
+
+#title {
+  padding-left: 1rem;
+  color: var(--color2);
+}
```
Comment (mwanawabangona) [inline styles/index.css]: - [ ] `required`
- [ ] Amazing job so far!! 👏 
- [x] However, the pages should look almost identical to the original design, 
- [x] The pages use the exact same `colors`, for `text`, `backgrounds`, `borders`, etc, from the original design
- [x] Please make the navbar spacing as the original template and reduce the main header
image just like the original design

- [ ] see screenshots below for comparisons

`your design`
headerrr

`figma template`
header shin

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @MarcoOquendoC ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @MarcoOquendoC 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by MarcoOquendoC** [comment]
# Hi @andersonlebon 
There is nothing else to say other than...
Thank youuuuuuuuuuuuuuuuuuuuuu
R

**Pull request merged**